XCUITests/TopTabsTest.swift

```diff
@@ -190,27 +190,31 @@ class TopTabsTest: BaseTestCase {
     func testLongTapTabCounter() {
         if !iPad() {
             // Long tap on Tab Counter should show the correct options
-            waitForExistence(app.buttons["Show Tabs"], timeout: 5)
+            waitForExistence(app.buttons["Show Tabs"], timeout: 10)
             app.buttons["Show Tabs"].press(forDuration: 1)
             waitForExistence(app.cells["quick_action_new_tab"])
             XCTAssertTrue(app.cells["quick_action_new_tab"].exists)
+            waitForExistence(app.cells["tab_close"])
```

Comment:
No need to add this `waitForExists`, it is waiting for an option shown in the same menu as the previous option and there is already a wait for it